fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -202,6 +203,13 @@ rocksdb::DBOptions SharedRocksDBState::initialDbOptions() {
 	options.statistics->set_stats_level(rocksdb::kExceptHistogramOrTimers);
 
 	options.db_log_dir = SERVER_KNOBS->LOG_DIRECTORY;
+
+	if (SERVER_KNOBS->ROCKSDB_MUTE_LOGS) {
+		options.info_log = std::make_shared<NullRocksDBLogForwarder>();
```

Comment:
Can we not leave it without passing anything to info_log?

Reply:
I suppose the purpose of this is to benchmark `RocksDBLogForwarder`?